### ISSUE TITLE
Session Start Error

### ISSUE THREAD

**Issue by MelusiZondi** [SafeExamBrowser/seb-win-refactoring#976]
> [!IMPORTANT]
> - Please _always_ consult the documentation first before creating a bug report: https://safeexambrowser.org/windows/win_usermanual_en.html.
> - Please _always_ attach the log file(s) of the affected session(s)! They can be found under `%LocalAppData%\SafeExamBrowser\Logs`.

**Describe the Bug**
SEB failed to start a new session! Please consult the log files for more information.

C:\Users\LibUser\AppData\Local\SafeExamBrowser\Logs\2024-09-17_14h50m17s_Client.log)

C:\Users\LibUser\AppData\Local\SafeExamBrowser\Logs\2024-09-17_14h50m17s_Runtime.log) 

**Steps to Reproduce**

**Expected Behavior**
SEB Launches successfully and runs the test/exams.

![image](https://github.com/user-attachments/assets/ced9d092-ea94-4aa6-8b9a-f052aa5cc20e)

**Version Information**
 System information;
Processor   11th Gen Intel(R) Core(TM) i7-1185G7 @ 3.00GHz   1.80 GHz
Installed RAM     8,00 GB (7,73 GB usable)
System type 64-bit operating system, x64-based processor
Edition     Windows 11 Pro
Version     22H2
Installed on      ‎2023/‎04/‎22
OS build    22621.3447
Experience  Windows Feature Experience Pack 1000.22688.1000.0

App Version 3.7.1 (x64)
**Additional Context**

**Comment by dbuechel** [comment]
Please provide the application logs of the affected session(s). They can be found under `%LocalAppData%\SafeExamBrowser\Logs`.

**Comment by MelusiZondi** [comment]
Hi Damian,

Please find attached log files as requested.

Regards

________________________________

**Comment by MelusiZondi** [comment]
Hi Damian, 

I did send the log files via email, however, please see attached below;-

[2024-09-18_09h40m55s_Client.log](https://github.com/user-attachments/files/17239764/2024-09-18_09h40m55s_Client.log)
[2024-09-18_09h40m55s_Runtime.log](https://github.com/user-attachments/files/17239765/2024-09-18_09h40m55s_Runtime.log)


Kindly assist soonest.

**Comment by dbuechel** [comment]
As the client log already reveals in plain English:

```
2024-09-18 09:41:01.994 [01] - INFO: Initializing applications...
2024-09-18 09:41:02.402 [01] - DEBUG: [ApplicationMonitor] Initialized 254 currently running processes.
2024-09-18 09:41:02.402 [01] - DEBUG: [ApplicationMonitor] Initialized blacklist with 49 applications: AA_v3.exe, AeroAdmin.exe, beamyourscreen-host.exe, CamPlay.exe, Camtasia.exe, CamtasiaStudio.exe, Camtasia_Studio.exe, CamRecorder.exe, CamtasiaUtl.exe, chromoting.exe, CiscoCollabHost.exe, CiscoWebExStart.exe, Discord.exe, Element.exe, g2mcomm.exe, g2mcomm.exe, g2mlauncher.exe, g2mstart.exe, GotoMeetingWinStore.exe, join.me.exe, join.me.sentinel.exe, Microsoft.Media.player.exe, Mikogo-host.exe, MS-teams.exe, obs32.exe, obs64.exe, PCMonitorSrv.exe, pcmontask.exe, ptoneclk.exe, RemotePCDesktop.exe, remoting_host.exe, RPCService.exe, RPCSuite.exe, Skype.exe, SkypeApp.exe, SkypeHost.exe, slack.exe, spotify.exe, SRServer.exe, strwinclt.exe, Teams.exe, TeamViewer.exe, Telegram.exe, VLC.exe, vncserver.exe, vncviewer.exe, vncserverui.exe, webexmta.exe, Zoom.exe
2024-09-18 09:41:02.402 [01] - DEBUG: [ApplicationMonitor] Process 'SRServer.exe' (10000) belongs to application 'SRServer.exe' and needs to be terminated.
2024-09-18 09:41:02.418 [01] - DEBUG: [ApplicationMonitor] Initialized whitelist with 0 applications.
2024-09-18 09:41:02.418 [01] - INFO: The following applications need to be terminated: SRServer.exe.
2024-09-18 09:41:04.019 [01] - INFO: The user chose to automatically terminate all running applications.
2024-09-18 09:41:04.019 [01] - DEBUG: [Process 'SRServer.exe' (10000)] Attempting to close process...
2024-09-18 09:41:04.019 [01] - WARNING: [Process 'SRServer.exe' (10000)] Failed to send close message to main window!
[...]
2024-09-18 09:41:04.035 [01] - DEBUG: [Process 'SRServer.exe' (10000)] Attempting to close process...
2024-09-18 09:41:04.035 [01] - WARNING: [Process 'SRServer.exe' (10000)] Failed to send close message to main window!
2024-09-18 09:41:04.067 [01] - ERROR: [Process 'SRServer.exe' (10000)] Failed to check whether process is terminated!

   Exception Message: Access is denied
   Exception Type: System.ComponentModel.Win32Exception

   at System.Diagnostics.ProcessManager.OpenProcess(Int32 processId, Int32 access, Boolean throwIfExited)
   at System.Diagnostics.Process.GetProcessHandle(Int32 access, Boolean throwIfExited)
   at System.Diagnostics.Process.get_HasExited()
   at SafeExamBrowser.WindowsApi.Processes.Process.IsTerminated() in C:\Users\appveyor\projects\seb-win-refactoring-mltg1wfbmuoivnbc\SafeExamBrowser.WindowsApi\Processes\Process.cs:line 126

2024-09-18 09:41:04.067 [01] - DEBUG: [Process 'SRServer.exe' (10000)] Attempting to kill process...
2024-09-18 09:41:04.082 [01] - ERROR: [Process 'SRServer.exe' (10000)] Failed to kill process!

   Exception Message: Access is denied
   Exception Type: System.ComponentModel.Win32Exception

   at System.Diagnostics.ProcessManager.OpenProcess(Int32 processId, Int32 access, Boolean throwIfExited)
   at System.Diagnostics.Process.GetProcessHandle(Int32 access, Boolean throwIfExited)
   at System.Diagnostics.Process.Kill()
   at SafeExamBrowser.WindowsApi.Processes.Process.TryKill(Int32 timeout_ms) in C:\Users\appveyor\projects\seb-win-refactoring-mltg1wfbmuoivnbc\SafeExamBrowser.WindowsApi\Processes\Process.cs:line 103

[...]

2024-09-18 09:41:04.098 [01] - ERROR: [Process 'SRServer.exe' (10000)] Failed to check whether process is terminated!

   Exception Message: Access is denied
   Exception Type: System.ComponentModel.Win32Exception

   at System.Diagnostics.ProcessManager.OpenProcess(Int32 processId, Int32 access, Boolean throwIfExited)
   at System.Diagnostics.Process.GetProcessHandle(Int32 access, Boolean throwIfExited)
   at System.Diagnostics.Process.get_HasExited()
   at SafeExamBrowser.WindowsApi.Processes.Process.IsTerminated() in C:\Users\appveyor\projects\seb-win-refactoring-mltg1wfbmuoivnbc\SafeExamBrowser.WindowsApi\Processes\Process.cs:line 126

2024-09-18 09:41:04.098 [01] - WARNING: [ApplicationMonitor] Failed to terminate process 'SRServer.exe' (10000)!
2024-09-18 09:41:04.098 [01] - ERROR: [Process 'SRServer.exe' (10000)] Failed to check whether process is terminated!

   Exception Message: Access is denied
   Exception Type: System.ComponentModel.Win32Exception

   at System.Diagnostics.ProcessManager.OpenProcess(Int32 processId, Int32 access, Boolean throwIfExited)
   at System.Diagnostics.Process.GetProcessHandle(Int32 access, Boolean throwIfExited)
   at System.Diagnostics.Process.get_HasExited()
   at SafeExamBrowser.WindowsApi.Processes.Process.IsTerminated() in C:\Users\appveyor\projects\seb-win-refactoring-mltg1wfbmuoivnbc\SafeExamBrowser.WindowsApi\Processes\Process.cs:line 126

2024-09-18 09:41:04.098 [01] - ERROR: Failed to automatically terminate application 'SRServer.exe'!
```

In particular: `Failed to automatically terminate application 'SRServer.exe'!`, i.e. you defined that application to be a prohibited application but SEB failed to automatically terminate it (potentially because it is running with administrator privileges or alike). Simply terminate it manually and restart SEB, or then investigate what the potential issue could be with the failed termination of that particular application.
